terraform/modules/elasticache/security_groups.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_security_group" "redis" {
+  name        = "${var.prefix}-redis-security-group"
+  description = "Redis security group"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description = "Allow ingress on port 6379 from anything within the private subnet cidr block range"
+    protocol    = "tcp"
+    from_port   = 6379
+    to_port     = 6379
+    cidr_blocks = [var.private_subnet_cidr]

```

Comment:
This might be better restricting on security group than on anything in our private subnet? (Although both would prevent external access which is the largest concern, I would normally expect restrictions to be done by security group except when being explicitly unrestrictive)

Reply:
I agree - now updated 👍